osu.Game/Overlays/Dialog/PopupDialogDangerousButton.cs

```diff
@@ -76,13 +75,13 @@ protected override void LoadComplete()
 
             protected override void AbortConfirm()
             {
-                lowPassFilter.CutoffTo(AudioFilter.MAX_LOWPASS_CUTOFF);
+                musicController?.Unduck();
                 base.AbortConfirm();
             }
 
             protected override void Confirm()
             {
-                lowPassFilter.CutoffTo(AudioFilter.MAX_LOWPASS_CUTOFF);
+                musicController?.Duck();
```

Comment:
In other cases `lowPassFilter.CutoffTo(...)` was replaced by `musicController.Duck(..., volume: 1f)` but the volume spec is missing here, which means that this is a difference in behaviour, as in volume will decrease *in addition to* the low pass filter. Was that the intention?